requirements-test.txt

```diff
@@ -0,0 +1,7 @@
+crash
+cratedb-toolkit==0.0.2
+pueblo[proc]==0.0.4
+prometheus-api-client<0.6
+pytest<8
+pytest-env<2
+requests<3
```

Comment:
Should we also add a corresponding section to the dependabot config, to keep the test requirements up to date?

Reply:
That was clearly an oversight, thank you so much for spotting that flaw. Fixed with f43b3d4c0b86d.